env.example

```diff
@@ -3,3 +3,9 @@ NODE_ENV=development
 
 # PORT is the port used by the web server
 PORT=8080
+
+# LOG_LEVEL is used to set the level of debugging for the logs.
+LOG_LEVEL=debug
+
+# LOG_FILE is used to set a destination path to write logs. Works in production mode only.
+LOG_FILE=./logFile.log
```

Comment:
Leave this blank.  Let's not include one by default.

```
LOG_LEVEL=
```

Reply:
Done.